app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_type/AddBusStopType.java

```diff
@@ -0,0 +1,52 @@
+package de.westnordost.streetcomplete.quests.bus_stop_type;
+
+import android.os.Bundle;
+
+import de.westnordost.streetcomplete.data.QuestImportance;
+import de.westnordost.streetcomplete.data.osm.OverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+
+public class AddBusStopType extends OverpassQuestType
+{
+	@Override
+	protected String getTagFilters()
+	{
+		return "nodes with public_transport=platform and !shelter and !informal";
```

Comment:
All of that makes sense, and I understand better what defines a quest (maybe this can be documented for developers wanting to create new quests ?). So we can have two quests : one for shelter, one for bench. Each one must be answered simply by yes/no. The wizard system for answering several quests over a same object seems a good compromise. Will you create an issue for that ?